addon/index.ts

```diff
@@ -88,8 +93,8 @@ export function initalizeLocalStorageKey(key) {
 //
 // Borrowed from the Ember Data source code:
 // https://github.com/emberjs/data/blob/22a8f20e2f11ed82c85160944e976073dc530d8b/packages/model/addon/-private/util.ts#L5
-function isElementDescriptor(...args) {
-  let [maybeTarget, maybeKey, maybeDescriptor] = args;
+function isElementDescriptor(...args: unknown[]) {
```

Comment:
Let's add a return type explicitly for clarity:

```suggestion
function isElementDescriptor(...args: unknown[]): boolean {
```